debian/control

```diff
@@ -62,7 +69,7 @@ Package: golang-doc
 Replaces: golang-weekly-doc
 Conflicts: golang-weekly-doc
 Provides: golang-weekly-doc
-Depends: ${misc:Depends}, golang-go
+Depends: golang-go, ${misc:Depends}
```

Comment:
Why this change (and other swaps like it below)? I think it might make sense to keep the delta minimal when introducing a new upstream version with so many changes, and maybe push the changes either as a separate debian version (`-2`), or with a new minor upstream release.